chart/kubeapps/templates/apprepositories-secret.yaml

```diff
@@ -4,6 +4,28 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: {{ template "kubeapps.apprepository-secret.name" . }}
+  {{- if .namespace }}
+  namespace: {{ .namespace }}
+  {{- end }}
+  annotations:
+    "helm.sh/hook": pre-install
+  labels:{{ include "kubeapps.labels" $ | nindent 4 }}
+data:
+  {{- if .caCert }}
+  ca.crt: |-
+    {{ .caCert | b64enc }}
+  {{- end }}
+  {{- if .authorizationHeader }}
+  authorizationHeader: |-
+    {{ .authorizationHeader | b64enc }}
+  {{- end }}
+---
+{{/* credentials are required in the release namespace for syncer jobs */}}
```

Comment:
Until we get time to tackle #1647 :)

Reply:
That's a nice issue, would simplify this quite a bit!

Would you like this linked in the code or is it fine as it is?